db/migrate/20170120164058_create_task_for_each_job_and_transfer_attributes.rb

```diff
@@ -9,6 +9,12 @@ class MiqTask < ActiveRecord::Base
   end
 
   def up
+    say_with_time("Deleting finished jobs older than 7 days") do
+      Job.where("updated_on < ?", Time.now.utc - 7.days).where("state = 'finished'").delete_all
+    end
+    say_with_time("Deleting finished tasks older than 7 days") do
+      MiqTask.where("updated_on < ?", Time.now.utc - 7.days).where("state = 'Finished'").delete_all
```

Comment:
Will any of these have associated logs or blobs
```ruby
class MiqTask
  has_one :log_file, :dependent => :destroy
  has_one :binary_blob, :as => :resource, :dependent => :destroy
  # ...
  has_one :job, :dependent => :destroy
end
```

Do we need something like
```ruby
LogFile.where(:miq_task_id => MiqTask.where("updated_on < ?", 7.days.ago.utc).select(:id)).delete_all
BinaryBlob.where(:resource_type => 'MiqTask', :resource_id => MiqTask.where("updated_on < ?", 7.days.ago.utc).select(:id)).delete_all
```

(code not tested)